lighthouse-extension/gulpfile.js

```diff
@@ -155,6 +162,33 @@ gulp.task('browserify', cb => {
   runSequence('browserify-lighthouse', 'browserify-other', cb);
 });
 
+gulp.task('compilejs', () => {
+  const opts = {
+    mangle: false, // Preserve  array notation property access. Needed for artifact name lookups.
```

Comment:
Oh right because we do use `Function.name` there with the gatherer class name, maybe update that comment then? The accessors seems like the secondary issue to the primary problem of the gatherer being renamed :)